pkgs/development/python-modules/libcst/default.nix

```diff
@@ -77,6 +77,8 @@ buildPythonPackage rec {
     "TypeInferenceProviderTest"
     # we'd need to run `python -m libcst.codegen.generate all` but shouldn't modify $out
     "test_codegen_clean_visitor_functions"
+    "test_codegen_clean_matcher_classes"
+    "test_codegen_clean_return_types"
```

Comment:
If I update libcst to 1.8.6 on master it builds without disabling these tests (on x86_64-linux).
What's the error message you get?